.rustfmt.toml

```diff
@@ -40,4 +40,4 @@ wrap_comments = true
 
 # Which version of the formatting rules to use. Version::One is backwards-compatible with Rustfmt 1.0. Other versions are only backwards compatible within a major version number.
 # Default value: "One"
-version = "Two"
+style_edition = "2024"
```

Comment:
Why did you change that ?

Reply:
![image](https://github.com/user-attachments/assets/cdf3a412-e85c-43c5-9cd8-68f440791b45)

Deprecated.

Also I suggest we simply commit a rust-toolchain file like done in findex server. I committed one, make me notice if you disagree with the idea to delete it